### ISSUE TITLE
Files/directories (from PKG-INFO) not found python 3.7

### ISSUE THREAD

**Issue by kankayaibrahim** [ibmdb/python-ibmdb#335]
Collecting ibm_db
  Using cached https://files.pythonhosted.org/packages/f8/6f/5f8186cb31021409235a948be5e29d77761ef92747a101360003747c4105/ibm_db-2.0.8.tar.gz

Files/directories (from PKG-INFO) not found in C:\Users\username\AppData\Local\Temp\pycharm-packaging\ibm-db\pip-egg-info

**Comment by SabaKauser** [comment]
clear you pip cache:
%LocalAppData%\pip\Cache
and retry install

**Comment by kankayaibrahim** [comment]
The same problem continues

**Comment by SabaKauser** [comment]
can you try 
pip install ibm_db==2.0.8a

**Comment by kankayaibrahim** [comment]
Collecting ibm_db==2.0.8a
  Downloading https://files.pythonhosted.org/packages/02/76/2362a41e360596bb544ee0a85192fe26c3291d2fa595876e04f700a8f7a2/ibm_db-2.0.8a.tar.gz (689kB)
    100% |████████████████████████████████| 696kB 1.9MB/s
Files/directories (from PKG-INFO) not found in C:\Users\username\AppData\Local\Temp\pip-install-gfp6na1f\ibm-db\pip-egg-info

I tested on 3 computers the same problem

**Comment by kankayaibrahim** [comment]
Successfully installed ibm-db-2.0.9 thanks

**Comment by SabaKauser** [comment]
closing.

**Comment by Lenbok** [comment]
What was the actual resolution to this? I'm having the same problem:

```
$ pip install ibm_db==2.0.9
Collecting ibm_db==2.0.9
  Downloading https://files.pythonhosted.org/packages/3f/61/389c6decacfed61c2724005b8c2be61f2c699f731b53af13517e5a0e6e98/ibm_db-2.0.9.tar.gz (782kB)
     |████████████████████████████████| 788kB 93kB/s 
ERROR: Files/directories not found in /tmp/pip-install-w1f9li_p/ibm-db/pip-egg-info
```
That was into a fresh python3 venv.

**Comment by ricdodds** [comment]
Having the same problem:
```
$ pip install ibm-db==3.0.1
Collecting ibm-db==3.0.1
  Using cached https://files.pythonhosted.org/packages/98/cb/f77d9bd5f64246074af364cc30e20e3044c533890f3b67d30e89615c2fc5/ibm_db-3.0.1.tar.gz
ERROR: Files/directories not found in /tmp/pip-install-t2teebgx/ibm-db/pip-egg-info
```
Any suggestions?

**Comment by Lenbok** [comment]
For future reference, the solution was to run pip in verbose mode which gives more information on the actual cause of the error. During installation, the module was looking at env vars set by an old db2 installation. Once I erased those vars I was able to proceed. (I think I manually installed the clidriver and set IBM_DB_HOME, rather than letting the pip install download it's own version, since I wanted to use it across multiple venvs)
